src/install.rst

```diff
@@ -24,328 +24,266 @@ The following instructions describe how to install Nextstrain's software tools,
 
-.. tabs::
-
-   .. group-tab:: macOS
-
-      .. note::
-
-         If you are an experienced user, you can replace ``conda`` with ``pip`` but :doc:`note the extra installation steps for Augur <augur:installation/installation>` and :doc:`install Auspice via npm <auspice:introduction/install>`.
-
-      1. Install Miniconda:
-
-         a. Go to the `installation page <https://docs.conda.io/en/latest/miniconda.html>`_.
-         b. Scroll down to the **Latest Miniconda Installer Links** section and click the MacOSX platform link that ends with **pkg**.
-         c. Open the downloaded file and follow through installation prompts.
-
-      2. Open a terminal window.
-      3. Install Mamba on the ``base`` Conda environment:
-
-         .. code-block:: bash
-
-            conda install -n base -c conda-forge mamba --yes
-            conda activate base
-
-      4. Install the Nextstrain components. There are two options:
-
-         .. tabs::
-
-            .. group-tab:: Docker (recommended)
-
-               .. warning::
-
-                  If using a newer Mac with an `Apple silicon chip <https://support.apple.com/en-us/HT211814>`_ (e.g. M1), **Native** installation is recommended due to slowness with the Docker installation. `We are considering ways to improve this <https://github.com/nextstrain/docker-base/issues/35>`_.
-
-               1. `Install Docker Desktop using the official guide <https://docs.docker.com/desktop/install/mac-install/>`_.
-               2. Create a Conda environment named ``nextstrain``:
-
-                  .. include:: snippets/conda-create-bash.rst
-
-               3. Install the Nextstrain CLI:
-
-                  .. code-block:: bash
-
-                     mamba install --yes nextstrain-cli
-
-            .. group-tab:: Native
+First, install the Nextstrain components. There are two options:
 
-               .. warning::
-
-                  If using a newer Mac with an `Apple silicon chip <https://support.apple.com/en-us/HT211814>`_ (e.g. M1), first run these commands to ensure Conda creates the environment with ``osx-64`` emulation:
-
-                  .. code-block:: bash
-
-                     # Create a new environment using Intel packages called base_osx-64
-                     CONDA_SUBDIR=osx-64 conda create -n base_osx-64 python
-
-                     # Activate new Intel-based environment
-                     conda activate base_osx-64
-
-                     # Ensure future Conda commands in this environment use Intel packages too.
-                     conda config --env --set subdir osx-64
-
-               1. Create a Conda environment named ``nextstrain``:
+.. tabs::
 
-                  .. include:: snippets/conda-create-bash.rst
+   .. group-tab:: Docker (recommended)
 
-               2. Install all the necessary software:
+      .. tabs::
 
-                  .. include:: snippets/conda-install-full-bash.rst
+         .. group-tab:: macOS
```

Comment:
Nice, I'm on board with this. Seems appropriate in this PR to add some text describing how to choose between Docker vs. Native (would supersede #106) since this decision on the user is very obvious from the top-level tabs.

Reply:
Indeed!

Reply:
[Relevant notes](https://docs.google.com/document/d/1dODYr-3T8LAQt-ewMZKNrcy1Wvy70Fw-h8exXbIwbyM/edit#heading=h.7nuap5p45ey7) (also linked from the linked PR).

Reply:
Added in 9f61ba8.